database/mssql/scripts/dbscripts/PSP_PIMS_LATEST/Alter Up/76_DML_PIMS_CLAIM.sql

```diff
@@ -0,0 +1,12 @@
+GO
+INSERT [dbo].[PIMS_CLAIM] ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
+VALUES (40, N'0db3f117-3da9-40bc-b5db-df676f40cc06', N'activity-add', N'0db3f117-3da9-40bc-b5db-df676f40cc06', N'Ability to add activities.', 0, 1, CAST(N'2022-07-15T02:17:49.903' AS DateTime), N'SEED', NULL, N'SEED', CAST(N'2022-07-15T07:42:25.790' AS DateTime), N'service', N'1b93f614-91da-4b32-b36e-bd2c6ebd12e2', N'', CAST(N'2022-07-15T02:17:49.903' AS DateTime), N'SEED', CAST(N'2022-07-15T07:42:25.780' AS DateTime), N'SEED')
+GO
+INSERT [dbo].[PIMS_CLAIM] ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
+VALUES (41, N'7ebcb98d-f2fb-4826-a8e3-14729b95cc95', N'activity-edit', N'7ebcb98d-f2fb-4826-a8e3-14729b95cc95', N'Ability to edit activities.', 0, 1, CAST(N'2022-07-15T02:17:49.903' AS DateTime), N'SEED', NULL, N'SEED', CAST(N'2022-07-15T07:42:25.890' AS DateTime), N'service', N'1b93f614-91da-4b32-b36e-bd2c6ebd12e2', N'', CAST(N'2022-07-15T02:17:49.903' AS DateTime), N'SEED', CAST(N'2022-07-15T07:42:25.873' AS DateTime), N'SEED')
+GO
+INSERT [dbo].[PIMS_CLAIM] ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID])
+VALUES (42, N'04f3a5e1-0423-4489-ab39-96c8f7801c29', N'activity-delete', N'04f3a5e1-0423-4489-ab39-96c8f7801c29', N'Ability to delete activities.', 0, 1, CAST(N'2022-07-15T02:17:49.903' AS DateTime), N'SEED', NULL, N'SEED', CAST(N'2022-07-15T07:42:25.990' AS DateTime), N'service', N'1b93f614-91da-4b32-b36e-bd2c6ebd12e2', N'', CAST(N'2022-07-15T02:17:49.903' AS DateTime), N'SEED', CAST(N'2022-07-15T07:42:25.967' AS DateTime), N'SEED')
+GO
```

Comment:
Same as the other scripts; Define the parameters instead of repeating them inline.

Reply:
@devinleighsmith this does not look addressed